package.json

```diff
@@ -82,21 +82,21 @@
   "scripts": {
     "ci": "yarn lint && yarn test",
     "clean-out": "rimraf ./out",
-    "vscode:prepublish": "yarn clean-out && tsc -p ./src",
-    "compile": "tsc -watch -p ./src",
+    "vscode:prepublish": "yarn clean-out && tsc -p ./tsconfig.publish.json",
+    "compile": "tsc -watch -p ./tsconfig.publish.json",
```

Comment:
Mainly because when the tsconfig was in a nested directory, I couldn't augment declarations.  So the extension of jest-editor-support.d.ts wasn't working properly with a nested tsconfig.